torchao/csrc/cuda/tensor_core_tiled_layout/tensor_core_tiled_layout.cu

```diff
@@ -7,13 +7,24 @@
 #include <c10/cuda/CUDAGuard.h>
 #include <torch/extension.h>
 
+#if defined(USE_ROCM)
+#include <hip/hip_bf16.h>
+#include <hip/hip_fp16.h>
+#include <hip/hip_runtime.h>
+#endif
+
 template <typename U, typename V>
 constexpr __host__ __device__ auto divUp(U a, V b) -> decltype(a + b) {
   static_assert(std::is_integral<U>::value && std::is_integral<V>::value, "");
   const uint64_t blocks = a / b + (a % b != 0);
   return blocks;
 }
+
+#if defined(USE_ROCM)
+constexpr int32_t kWarpSize = 64;
```

Comment:
not a rocm expert but how come? is this gpu specific? Worth adding a quick comment

Reply:
Yes, all instict compute on ROCm uses warp size 64.

Reply:
@msaroufim We do have a plan to support device query and different arch in later PRs. Currently, AMD CDNA (MI300x) use warp size 64, while RDNA (NAVI) use warp size 32.

Other than performance implication, one possible issue related to warp size would be in-warp reduction result.